news/atom.xml

```diff
@@ -14,8 +14,11 @@ layout:
     <name>The ev3dev team</name>
   </author>
 
+  {% assign index = 0 %}
   {% for post in site.posts %}
```

Comment:
Or even better `{% for post in site.posts limit:10 %}

https://github.com/shopify/liquid/wiki/Liquid-for-Designers#helper-variables